relay-common/src/glob.rs

```diff
@@ -89,12 +89,13 @@ fn translate_codeowners_pattern(pattern: &str) -> Option<Regex> {
                 let right_anchored = i + 2 == pattern.len();
                 let trailing_slash =
                     i + 2 < pattern.len() && pattern.chars().nth(i + 2) == Some('/');
-                let leading_star_slash = pattern.chars().nth(i + 2) == Some('/');
+                let star_star_slash = pattern.chars().nth(i + 1) == Some('*')
```

Comment:
One thing I missed in my previous review: `pattern.chars().nth()` has a complexity of `O(n)`, because it has to go through the string and parse UTF-8 every time. So it would be better to `.collect()` the chars into a `Vec` on line 68 and iterate over that, so you can access neighboring chars in `O(1)`.

Reply:
Thank you for your suggestion. I have changed this